src/main/java/hu/oe/nik/szfmv/automatedcar/model/ParkingSpace.java

```diff
@@ -3,8 +3,45 @@
 import hu.oe.nik.szfmv.automatedcar.model.interfaces.IBackground;
 import hu.oe.nik.szfmv.automatedcar.model.interfaces.IStatic;
 
+import javax.xml.bind.Unmarshaller;
+import java.awt.Rectangle;
+
 /**
  * Parkolóhely alaposztály.
  */
 public class ParkingSpace extends WorldObject implements IStatic, IBackground {
+    private final int LEFT_BORDER = 4;
+    private final int RIGHT_BORDER = 14;
+    private final int TOP_BORDER = 10;
+    private final short BOTTOM_BORDER = 4;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void afterUnmarshal(Unmarshaller u, Object parent) {
+        super.afterUnmarshal(u, parent);
+        this.position.setZ(1);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initShape(){
+        switch (this.imageFileName) {
+            case "parking_space_parallel":
```

Comment:
Ezekre konstans kellene (minden osztályban). A NetBeans tud olyat, hogy nyomsz egy jobb klikket a kijelölt stringen és ki tudod mozgatni konstansba. Szerintem az IDEA is tudja, ciki lenne ha nem. Ha lehet, akkor rögtön mehetne a Consts osztályba. Ha nem, akkor először menjen ki a Road osztályba, aztán onnan jobb klikk, move (vagy vmi hasonló) és mehet a consts-ba. (lustaság kimaxolva 😄)